void function should not return a value in wrappers

Steps to reproduce:

- Open `coreutils` project
- Choose `wc` target
- Generate tests for `src/wc.c`
- Run tests
- See error in logs
```
/home/utbot/projects/coreutils/utbot/wrapper/gnulib/lib/malloc/dynarray_at_failure_wrapper.c:9:1: error: void function 'gl_dynarray_at_failure_gnulib_lib_malloc_dynarray_at_failure_c' should not return a value [-Wreturn-type]
return gl_dynarray_at_failure(size, index);
```

--- a/server/src/clang-utils/SourceToHeaderMatchCallback.cpp
+++ b/server/src/clang-utils/SourceToHeaderMatchCallback.cpp
@@ -289,7 +289,10 @@ void SourceToHeaderMatchCallback::printReturn(const FunctionDecl *decl,
     printer::Printer printer;
     auto args = CollectionUtils::transformTo<std::vector<std::string>>(
         decl->parameters(), [](ParmVarDecl *param) { return param->getNameAsString(); });
-    printer.ss << "return ";
+    bool noReturn = decl->isNoReturn() || decl->getReturnType()->isVoidType();
+    if (!noReturn) {
+        printer.ss << "return ";
+    }
     printer.strFunctionCall(name, args);
 
     *stream << printer.ss.str();

--- a/server/src/clang-utils/SourceToHeaderMatchCallback.cpp
+++ b/server/src/clang-utils/SourceToHeaderMatchCallback.cpp
@@ -144,7 +144,7 @@ void SourceToHeaderMatchCallback::handleTypedef(const TypedefDecl *decl) {
     auto canonicalType = decl->getUnderlyingType().getCanonicalType();
     auto name = decl->getName().str();
     auto canonicalName = canonicalType.getAsString();
-    if (name == "wchar_t") {
+    if (name == "wchar_t" && !forStubHeader) {
         // wchar_t is builtin type in C++ but is typedef in C
         return;
     }

